scripts/start.js

```diff
@@ -364,7 +369,8 @@ function getUserInfo () {
         resolve({
           userName: username.slice(0, username.indexOf('@')),
           ssoToken: body.access_token,
-          domain: domain
+          domain: domain,
+          userId: userId,
```

Comment:
@bond95 , please describe&verify what happens if the userId is missing in the userInfo.